R/verb-fill.R

```diff
@@ -200,4 +203,9 @@ last_value_sql.Hive <- function(con, x) {
   translate_sql(last_value(!!x, TRUE), con = con)
 }
 
+#' @export
+`last_value_sql.Microsoft SQL Server` <- function(con, x) {
+  build_sql("LAST_VALUE(", ident(as.character(x)), ") IGNORE NULLS", con = con)
```

Comment:
Does this need parentheses just to be sure?

```suggestion
  build_sql("(LAST_VALUE(", ident(as.character(x)), ") IGNORE NULLS)", con = con)
```

Reply:
I don't think so but we can also add some. `last_value_sql()` is only used in `fill()` and then the queries are simple select statements, e.g.

```SQL
SELECT `id`, `group`, LAST_VALUE(`n1` IGNORE NULLS) OVER (ORDER BY -`id`) AS `n1`
      FROM `df`
```

If you think it is still better to add parentheses please add them.